lib/AzureHandler/src/AzureHandler.cpp

```diff
@@ -0,0 +1,421 @@
+/*
```

Comment:
Nicely done!
Did you consider using [azure-sdk-for-c-arduino](https://github.com/Azure/azure-sdk-for-c-arduino)? 
Do you think it is not helping much?